README.adoc

```diff
@@ -91,13 +91,13 @@ tekton-dashboard   tekton-dashboard-tekton-pipelines.apps.kabanero.os.fyre.ibm.c
 Use the Jenkinsfile template from the reference section of this guide, and change the Docker Hub source to your own image repository. Push this Jenkinsfile to your GitHub repository in the root location.
 
 . **Create a deploy file**:
-On your local environment where Appsody is installed, go to your project folder and run the following command to generate an `app-deploy.yaml` file:
+On your local environment where your development application is installed, go to your project folder and run the following command to generate an `app-deploy.yaml` file:
 +
 ----
-appsody-deploy –-generate-only
+appsody build
```

Comment:
This is the current command which will create or update the app-deploy.yaml and replaces the now deprecated --generate-only.